GeneratorPgn.py

```diff
@@ -0,0 +1,318 @@
+import chess
+import torch.multiprocessing as mp
+import bz2
+from collections import deque
+import random
+import urllib
+import numpy as np
+
+class GeneratorPgn():

```

Comment:
if it generates PGN, it should be called PGNGenerator

Reply:
BTW, it actually read files, why not call it PGNReader?

Reply:
I think FENGenerator would be a better name, because in total this class recycle already modified pgn